dbmanager/dbmanager.go

```diff
@@ -0,0 +1,116 @@
+package dbmanager
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	_ "github.com/lib/pq" //db driver for postgres
+	"golang.org/x/crypto/bcrypt"
+
+)
+
+//////// QUERIES
+const (
+	userExistsQuery  = `SELECT uid from users WHERE email = $1`
+	getUserInfoQuery = `SELECT * from users WHERE email = $1`
+	getUserPassQuery = `SELECT password from users WHERE email = $1`
+	createUserQuery  = `INSERT INTO users(uid,email,password) VALUES (DEFAULT, $1 , $2);`
```

Comment:
I'm not 100% sure if you need to insert `DEFAULT` into `uid`. You might be able to omit that.

Reply:
Tested it and you are right. Removed